src/ray/core_worker/reference_count.cc

```diff
@@ -239,6 +239,36 @@ void ReferenceCounter::AddDynamicReturn(const ObjectID &object_id,
   AddNestedObjectIdsInternal(generator_id, {object_id}, owner_address);
 }
 
+void ReferenceCounter::OwnDynamicStreamingTaskReturnRef(const ObjectID &object_id,
```

Comment:
any test for this function? i can't find it.

Reply:
Will write one. It was also implicitly tested from task_manager_test.cc. But I will make another test in reference_counter.cc